src/index.js

```diff
@@ -38,6 +38,9 @@ export default function Knex(config) {
 // Expose Client on the main Knex namespace.
 Knex.Client = Client
 
+// Expose Knex version on the main Knex namespace.
+Knex.VERSION = require('../package.json').version
+
```

Comment:
Should we then change setting the version information on a knex instance to get its value from here>